src/abstractdataframe/abstractdataframe.jl

```diff
@@ -15,7 +15,7 @@ The following are normally implemented for AbstractDataFrames:
 * [`dump`](@ref) : show structure
 * `hcat` : horizontal concatenation
 * `vcat` : vertical concatenation
-* `repeat` : repeat rows
+* [`repeat`](@ref) : repeat data frame
```

Comment:
"repeat rows" was better IMHO (we already know it applies to data frames, the ambiguity is about rows vs columns).